static/app/views/automations/detail.tsx

```diff
@@ -49,7 +49,9 @@ function AutomationDetailContent({automation}: {automation: Automation}) {
   const location = useLocation();
   const navigate = useNavigate();
 
-  const {data: createdByUser} = useUserFromId({id: Number(automation.createdBy)});
+  const {data: createdByUser} = useUserFromId({
+    id: automation.createdBy ? Number(automation.createdBy) : undefined,
```

Comment:
if the `id` is null, should we make this request? (we can't just wrap with an if statement because this is using a hook, but we could halt the API request in the handler if the id is undefined)